tests/integration/components/mutually_exclusive/test_month_year_date_single_checkbox_override.py

```diff
@@ -0,0 +1,52 @@
+from tests.integration.components.mutually_exclusive.schema_urls import MUTUALLY_EXCLUSIVE_MONTH_YEAR_DATE
+
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestMonthYearDateSingleCheckboxOverride(IntegrationTestCase):
+    """
+    Tests to ensure that the server-side validation for mutually exclusive month year
+    with single checkbox override function as expected. These tests emulate the non-JS version.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey('test', 'mutually_exclusive_single_checkbox_override')
+        self.get(MUTUALLY_EXCLUSIVE_MONTH_YEAR_DATE)
+
+    def test_non_exclusive_answer(self):
+        # When
+        self.post({'month-year-date-answer-month': '10',
+                   'month-year-date-answer-year': '2018'})
+
+        # Then
+        self.assertInUrl('section-summary')
+        self.assertInPage('October 2018')
+
+    def test_exclusive_answer(self):
+        # When
+        self.post({'month-year-date-answer-month': '',
+                   'month-year-date-answer-year': '',
+                   'month-year-date-exclusive-answer': ['I prefer not to say']})
+
+        # Then
+        # self.assertInUrl('section-summary')
```

Comment:
Can this be removed? (and all the others in the same position)

Reply:
Same as above.